lisa/sut_orchestrator/azure/common.py

```diff
@@ -132,8 +139,12 @@ class AzureNodeSchema:
     shared_gallery_raw: Optional[Union[Dict[Any, Any], str]] = field(
         default=None, metadata=field_metadata(data_key="shared_gallery")
     )
-    vhd: str = ""
-    vmgs: str = ""
+    vhd_raw: Optional[Union[Dict[Any, Any], str]] = field(
+        default=None, metadata=field_metadata(data_key="vhd")
+    )
+    vmgs_raw: Optional[str] = field(
```

Comment:
vmgs is new thing, so it doesn't need to be a string in Azure Node level. It should be in vhd object.